package.json

```diff
@@ -50,6 +50,7 @@
 		"babel-preset-env": "^1.0.0",
 		"babel-register": "^6.0.0",
 		"chai": "^4.0.0",
+		"chai-as-promised": "7.1.1",
```

Comment:
Change this to `7.0.0` so that minor and patch updates are applied.

Reply:
added caret